canal/canal.go

```diff
@@ -199,18 +207,57 @@ func (c *Canal) GetTable(db string, table string) (*schema.Table, error) {
 		return t, nil
 	}
 
+	if c.cfg.DiscardNoMetaRowEvent {
+		c.tableLock.RLock()
+		lastTime, ok := c.errorTablesGetTime[key]
+		c.tableLock.RUnlock()
+		if ok && time.Now().Sub(lastTime) < UnknowTableRetryIntervalSec {
+			return nil, schema.LastGetTableInfoErr
+		}
+	}
+
 	t, err := schema.NewTable(c, db, table)
 	if err != nil {
 		// check table not exists
 		if ok, err1 := schema.IsTableExist(c, db, table); err1 == nil && !ok {
 			return nil, schema.ErrTableNotExist
 		}
-
-		return nil, errors.Trace(err)
+		// work around : RDSHeartBeat
+		// https://github.com/alibaba/canal/blob/master/parse/src/main/java/com/alibaba/otter/canal/parse/inbound/mysql/dbsync/LogEventConvert.java#L385
+		if key == schema.HAHealthCheckSchema {
```

Comment:
why do we need to check this schema if we meet any error?

Reply:
This is a common error in RDS, so I try to mock a table meta.
If we just skip and log error,  because RDS HAHeartBeat interval is very short, so we will have too many  HAHeartBeat errors in log files.
Should I add another config item, RDSFlag ?

Reply:
no need now, but I suggest adding more comments to explain detailedly here.

Reply:
done